src/main.h

```diff
@@ -237,7 +241,7 @@ void PruneAndFlush();
 
 /** (try to) add transaction to memory pool **/
 bool AcceptToMemoryPool(CTxMemPool& pool, CValidationState &state, const CTransaction &tx, bool fLimitFree,
-                        bool* pfMissingInputs, bool fOverrideMempoolLimit=false, bool fRejectAbsurdFee=false);
+                        bool* pfMissingInputs, bool fOverrideMempoolLimit=false, CAmount nAbsurdFee=0);
```

Comment:
I wouldn't oppose to just make nAbsurdFee mandatory (instead of defaulting to zero). In any case, we can ```const CAmount nAbsurdFee``` or ```const CAmount& nAbsurdFee```.